coralproject/talk
# [CORL-1296] New Comment Count

## What does this PR do?

With the `NEW_COMMENT_COUNT` feature flag, comment streams viewed will record the current comment count. When used in conjunction with [`count.js`](https://docs.coralproject.net/coral/v5/integrating/counts/), it will show `/ 5 New` when the number of comments exceeds what was previously recorded.



## What changes to the GraphQL/Database Schema does this PR introduce?

- Added `FEATURE_FLAG.NEW_COMMENT_COUNT`



## How do I test this PR?

1. Enable the `NEW_COMMENT_COUNT` feature flag.
2. View comments on a story.
3. In another browser, add a comment to that story.
4. In the first browser, view the comment count widget and notice the new `/ 1 New` added to the comment count text.
